Source/WebCore/platform/graphics/FontPlatformData.h

```diff
@@ -122,7 +122,7 @@ class FontPlatformData {
 #endif
 
 #if USE(FREETYPE)
-    FontPlatformData(cairo_font_face_t*, RefPtr<FcPattern>&&, float size, bool fixedWidth, bool syntheticBold, bool syntheticOblique, FontOrientation);
+    FontPlatformData(cairo_font_face_t*, RefPtr<FcPattern>&&, float size, bool fixedWidth, bool syntheticBold, bool syntheticOblique, FontOrientation, const FontCustomPlatformData* = nullptr);
```

Comment:
Bonus points if you change this to use FontCustomPlatformData&& or anything other than a raw pointer or reference.